### PR TITLE
Update libs "clipboard" and "marked"

### PR DESCRIPTION
**Pull request by rebeccawong7**
Three small changes which contribute towards optimizations listed in #331 

- Async load Google Maps JS 
- Update 3rd party libs: clipboard + marked
- Remove unused markdown component

### CODE REVIEW

File: src/views/pages/network.html [hunk 1]
```diff
@@ -21,22 +21,6 @@
 		{{/each}}
 	</style>
 
-	{{#if !@shared.isServer && !@shared.googleMapsLoaded}}
```
Comment (MartinKolarik) [inline src/views/pages/network.html:24]: There's no difference here, the scripts are already dynamically inserted and therefore async. Please revert this part.

File: src/public/js/decorators/clipboard.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module.exports = (node, title = 'Copy to Clipboard', tooltipPlacement = 'top', nodeSelector) => {
-	let clipboard = new Clipboard(node);
+	let clipboard = new ClipboardJS(node);
```
Comment (MartinKolarik) [inline src/public/js/decorators/clipboard.js:2]: This needs an update in the eslint config.

File: src/views/pages/network.html [hunk 3]
```diff
@@ -109,10 +109,10 @@ <h3>Big network in China</h3>
 
 				<div class="china-banner">
 					<div class="china-bg">
-						<img width="1140" height="328" src="{{@shared.assetsHost}}/img/network/china-bg.jpg" srcset="{{@shared.assetsHost}}/img/network/china-bg@2x.jpg 2x" loading="lazy">
```
Comment (MartinKolarik) [inline src/views/pages/network.html:112]: This seems to be a mistake?

File: src/public/js/decorators/clipboard.js (hunk 2 above)
Comment (rebeccawong7) [reply src/public/js/decorators/clipboard.js:2]: Sorry, should have caught this before submitting the PR

File: src/views/pages/network.html (hunk 1 above)
Comment (rebeccawong7) [reply src/views/pages/network.html]: Done! Out of curiosity where is the dynamic insertion happening from?

File: src/views/pages/network.html (hunk 3 above)
Comment (rebeccawong7) [reply src/views/pages/network.html]: Did one too many ctrl+z, reverted :)

File: src/views/pages/network.html (hunk 1 above)
Comment (MartinKolarik) [reply src/views/pages/network.html]: The whole file is a Ractive component so the rendering is handled by Ractive and this section is specifically restricted from server rendering (`!@shared.isServer`) so it happens on the client.

**Comment by MartinKolarik** [comment]
Looks good now 👍

File: src/views/pages/network.html (hunk 1 above)
Comment (rebeccawong7) [reply src/views/pages/network.html]: Thank you! I'll look into how Mustaches is used, I haven't come across it before

**Pull request merged**